### PR TITLE
Configured docker for run 8 processes in the uwsgi.

### PR DESCRIPTION
**Pull request by Zeeerooo**

### CODE REVIEW

File: adkintunmobile/server/uwsgi/Dockerfile [hunk 1]
```diff
@@ -41,4 +41,4 @@ RUN cd /adk/AdkintunMobile-Server && \
 ## Run command
 #run uwsgi
 CMD cd /adk/AdkintunMobile-Server && \
-	uwsgi --socket 0.0.0.0:8000  -w app:app
+	uwsgi --socket 0.0.0.0:8000  --processes 8 -w app:app
```
Comment (fmontoto) [inline adkintunmobile/server/uwsgi/Dockerfile]: Why 8?

**Comment by camilog** [comment]
LGTM

**Comment by fmontoto** [comment]
LGTM

**Comment by sebablasko** [comment]
Absolutely awesome, congratulations!

**Pull request merged**